util/fibers/detail/scheduler.cc

```diff
@@ -521,24 +558,22 @@ auto Scheduler::RunWorkerFibersStepImpl() -> RunFiberResult {
     fi->SwitchTo();
 
     ProactorBase::UpdateMonotonicTime();
-    uint64_t now_ns = ProactorBase::GetMonotonicTimeNs();
+    ran_ns = ProactorBase::GetMonotonicTimeNs() - start_ns;
 
     // It could be that we iterate over multiple fibers and meanwhile we postpone the execution of
     // brief tasks or do not handle I/O events. Therefore we limit the time spent in the loop
     // to kMaxTimeSpentNs.
-    if (now_ns > deadline_ns) {
```

Comment:
why you did not like `deadline_ns` ? it saved me a substract for each iteration :)